test/dialogmote/DialogmoteInnkallingSkjemaTest.tsx

```diff
@@ -397,6 +375,20 @@ describe("DialogmoteInnkallingSkjema", () => {
   });
 });
 
+const mountDialogmoteInnkallingSkjema = (mockStore: MockStore<unknown>) => {
+  return mount(
+    <MemoryRouter initialEntries={["/sykefravaer/dialogmote"]}>
+      <Route path="/sykefravaer/dialogmote">
```

Comment:
Om du gidder så kan du legge inn dette i en variabel ovenfor return siden den brukes to ganger:
`"/sykefravaer/dialogmote"`
Kan vurdere noe lignende i Avlyse-testen og på path. Kanskje hente "/sykefravaer/dialogmote" fra en ny variabel i Approuter?

Reply:
Fikset!